src/RoomsSDKAdapter.integration.test.js

```diff
@@ -0,0 +1,46 @@
+import {removeTestUser} from '@webex/test-users';
+
+import {createIntegrationTestUser, createIntegrationSdkInstance} from './testHelper';
+import RoomsSDKAdapter from './RoomsSDKAdapter';
+
+describe('Rooms SDK Adapter', () => {
+  let roomsSDKAdapter;
```

Comment:
In other repos ([components](https://github.com/webex/components/blob/master/src/adapters/ActivitiesJSONAdapter.test.js#L5), [interfaces](https://github.com/webex/component-adapter-interfaces/blob/master/src/WebexAdapter.test.js#L4)) we have variables in a single line